### DISCUSSION TITLE
#0 /home/offmyyar/public_html/backend/vendor/laravel/framework/src/Illuminate/Routing/Middleware/ThrottleRequests.php(133): Illuminate\Routing\Middleware\ThrottleRequests->addHeaders(NULL, 60, 55)

### DISCUSSION THREAD

**Discussion by Akposieyefa** [laravel/framework#38715] [Q&A]
- Laravel Version: 8
- PHP Version: 7.4
- Database Driver & Version:

### 500 Page isn't working 

getting this error in my Laravel 8 Project hosted on cPanel the project was working fine all of a sudden i started getting 500 error when i checked the log this is what i got i am confuse

**Comment by lk77** [comment]
this is unrelated to your 500 error
